test/routable_component_routes.js

```diff
@@ -63,31 +63,31 @@ describe('RoutableComponentRoutes', function() {
   });
   /** @test {RoutableComponentRoutes#constructor} */
   context('constructed with', function() {
-    subject(function() { return new RoutableComponentRoutes(this.routing_classes) });
+    subject(function() { return new RoutableComponentRoutes(this.routingClasses) });
```

Comment:
__[eslint]__
* Error - 'subject' is not defined. (no-undef)
* Error - Unexpected 'this'. (no-invalid-this)
* Error - Missing semicolon. (semi)

<!-- Comment by [SideCI](https://sideci.com) -->